internal/cmd/functional-benchmarker/main.go

```diff
@@ -1,34 +1,36 @@
 package main
 
 import (
+	"bufio"
 	"encoding/json"
 	"flag"
 	"fmt"
+	"io/ioutil"
 	"os"
-	"sort"
 	"time"
 
 	"github.com/ViaQ/logerr/log"
 
-	logging "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
 	"github.com/openshift/cluster-logging-operator/pkg/constants"
 	"github.com/openshift/cluster-logging-operator/pkg/utils"
 	"github.com/openshift/cluster-logging-operator/test"
-	"github.com/openshift/cluster-logging-operator/test/client"
-	"github.com/openshift/cluster-logging-operator/test/functional"
 	"github.com/openshift/cluster-logging-operator/test/helpers/types"
-	"github.com/openshift/cluster-logging-operator/test/runtime"
 )
 
 // HACK - This command is for development use only
 func main() {
 
 	image := flag.String("image", "quay.io/openshift/origin-logging-fluentd:latest", "The image to use to run the benchmark")
-	totalMessages := flag.Uint64("totMessages", 10000, "The number of messages to write")
-	msgSize := flag.Uint64("size", 1024, "The message size in bytes")
+	totalMessages := flag.Int("totMessages", 10000, "The number of messages to write per stressor")
+	msgSize := flag.Int("size", 1024, "The message size in bytes")
 	verbosity := flag.Int("verbosity", 0, "")
 	doCleanup := flag.Bool("docleanup", true, "set to false to preserve the namespace")
 	sample := flag.Bool("sample", false, "set to true to dump a sample message")
+	platform := flag.String("platform", "cluster", "The runtime environment: cluster (default), local (experimental). local requires podman")
+	output := flag.String("output", "default", "The output format: default, csv")
+
+	totLogStressors := flag.Int("exp-tot-stressors", 1, "Experimental. Total log stressors (platform=local)")
```

Comment:
can we change totLogStressors -> totalLogStressors and exp-tot-stressors->exp-total-stressors